src/features/events/components/appPromotionCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { colors } from '../../../theme/colors'
+import { mediaQueries } from '../../../theme/mediaQueries'
+import appleStore from '../../../theme/assets/images/apple.svg'
+import googlePlayStore from '../../../theme/assets/images/google-play.svg'
+
+const CardWrapper = styled.div`
+  padding-left: 20px;
+  margin-bottom: 40px;
+  ${mediaQueries.md} {
+    padding-left: 30px;
+    margin-bottom: 0;
+  }
+`
+
+const CardTitle = styled.h3`
+  margin: 40px 0 0.5em 0;
+  font-family: Poppins;
+  font-size: 2rem;
+  line-height: 1.19;
+  font-weight: bold;
+  color: ${colors.white};
+
+  ${mediaQueries.md} {
+    font-size: 2rem;
+    max-width: 174px;
+  }
+`
+
+const CardDescription = styled.p`
+  color: ${colors.white};
+  padding-right: 30px;
+  font-size: 1.125rem;
+  line-height: 26px;
+`
+const LinkWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  max-width: 300px;
+  ${mediaQueries.md} {
+    flex-direction: column;
+  }
+`
+
+const AppStoreLink = styled.a`
+  text-decoration: none;
+  margin: 5px 0;
+  ${mediaQueries.md} {
+    margin-right: 0;
+  }
+`
+const AppStoreImg = styled.img`
+  background: black;
+  width: 140px;
+  height: 48px;
+  object-fit: contain;
+`
+
+const AppPromotionCard: React.FC = () => {
+  return (
+    <CardWrapper>
+      <CardTitle>Download our app</CardTitle>
+      <CardDescription>
+        Download our iOS & Android app to carry our interactive Parade map,
+        stage line up info and afterparty information around with you on the day
+      </CardDescription>
+      <LinkWrapper>
+        <AppStoreLink
```

Comment:
Links with images inside them don't have any meaningful text to communicate to a user with a screen reader. Add an `aria-label` here